restx-specs-tests/src/test/webapp/hello.txt

```diff
@@ -0,0 +1 @@
+Hello world !
```

Comment:
I'm not sure about why you added `src/test/webapp/hello.txt` file.

To my POV, it pulls a lot of unnecessary complexity (but I can miss something related to Windows difference in behaviour).

To give you some background, `ResourcesRouteTest` is supposed to test `MyRestxResourcesRoute` which will serve any content below `/web/*` into the `classpath:static/*` file hierarchy
For example, if you have `classpath:static/hello.txt` (which is the case since we have `src/test/resources/static/hello.txt`), you will be able to access it through `GET /web/hello.txt` thanks to `MyRestxResourcesRoute`

Here, you're changing a lot of things :
- You're expecting to serve `hello.txt` from `src/test/webapp` instead of `src/test/resources`, making additionnal `pom.xml` to add `src/test/webapp` to `test-resources` (not that this changes leads to some test failures in `restx.md.RestxModuleDescriptorsDiscrepanciesDetectorWithPomAndIvyTest`
- You're commenting out assertions in `ResourcesRouteTest.java`

Not sure if this was really what you were expecting here :-)

Reply:
I didn’t see this hello file in src/resources/static
I agree I did two redundant things here. I added the file as well as the reference in the Pom. The thing is in the declaration src/test/webapp was not on the classpath because it is not referenced from the pom. Eclipse basically ignores it if it is not in the pom. It will work from a command line but not if u run the tests from eclipse. Do you want me to clean this up?

Reply:
`src/test/webapp` is never supposed to be added to the classpath (except if explicitely added to the pom)

Yes, you can go ahead an remove all stuff related to `src/test/webapp` :

- [ ] Remove additions to `pom.xml`

- [ ] Remove `src/test/webapp/hello.txt`

- [ ] Maybe uncomment commented code on `ResourcesRouteTest` as I guess it should work on windows as it is

Reply:
This was reverted by the latest push

Reply:
👌 
FYI, I executed locally (on a Mac) the full restx test suite and it passed successfully (it wasn't the case yesterday because of `RestxModuleDescriptorsDiscrepanciesDetectorWithPomAndIvyTest` failure)

So, currently, you didn't injected any regression in the test suite with your changes.

I'm still concerned about commented code and `replaceNewLine()` calls ;-)